run.sh

```diff
@@ -5,8 +5,10 @@ set -v -e -x
 
 export PATH=$PATH:/home/worker/bin
 
-if test "$PROCESSED_PACKAGES"; then
-  curl "$PROCESSED_PACKAGES" | gzip -dc > processed-packages
```

Comment:
I'd like to keep the ability to download it since that will make it easier to run this as a scheduled job in taskcluster. Having a copy checked in seems fine though.